src/main/java/net/minecraftforge/event/RegisterCommandsEvent.java

```diff
@@ -40,4 +43,9 @@ public Commands.CommandSelection getEnvironment()
     {
         return environment;
     }
+
+    public CommandBuildContext getContext()
```

Comment:
Could it be `getBuildContext()` instead? Also, could copy over the javadocs from the client commands event into here as well.